### PR TITLE
Add Spyglass 22.5.15

### PR DESCRIPTION
**Pull request by oparaskos**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/61989)

### CODE REVIEW

File: manifests/a/a5huynh/spyglass/2022.5.15/a5huynh.spyglass.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.7-2-4
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: a5huynh.spyglass
+PackageVersion: 2022.5.15
+PackageLocale: en-US
+Publisher: Andrew Huynh
```
Comment (ItzLevvie) [inline manifests/a/a5huynh/spyglass/2022.5.15/a5huynh.spyglass.locale.en-US.yaml:7]:
```suggestion
Publisher: athlabs
```
This must match Control Panel.

File: manifests/a/a5huynh/spyglass/2022.5.15/a5huynh.spyglass.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.7-2-4
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: a5huynh.spyglass
+PackageVersion: 2022.5.15
```
Comment (ItzLevvie) [inline manifests/a/a5huynh/spyglass/2022.5.15/a5huynh.spyglass.installer.yaml:5]:
```suggestion
PackageVersion: 22.5.15
```
This must match Control Panel.

File: manifests/a/a5huynh/spyglass/2022.5.15/a5huynh.spyglass.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.7-2-4
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: a5huynh.spyglass
+PackageVersion: 2022.5.15
```
Comment (ItzLevvie) [inline manifests/a/a5huynh/spyglass/2022.5.15/a5huynh.spyglass.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 22.5.15
```
This must match Control Panel.

File: manifests/a/a5huynh/spyglass/2022.5.15/a5huynh.spyglass.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.7-2-4
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: a5huynh.spyglass
+PackageVersion: 2022.5.15
```
Comment (ItzLevvie) [inline manifests/a/a5huynh/spyglass/2022.5.15/a5huynh.spyglass.yaml:5]:
```suggestion
PackageVersion: 22.5.15
```
This must match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/a/a5huynh/spyglass/2022.5.15/a5huynh.spyglass.yaml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.7-2-4
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: a5huynh.spyglass
```
Comment (KaranKad) [inline manifests/a/a5huynh/spyglass/2022.5.15/a5huynh.spyglass.yaml:4]:
```suggestion
PackageIdentifier: athlabs.spyglass
```

File: manifests/a/a5huynh/spyglass/2022.5.15/a5huynh.spyglass.installer.yaml [hunk 6]
```diff
@@ -0,0 +1,20 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.7-2-4
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: a5huynh.spyglass
```
Comment (KaranKad) [inline manifests/a/a5huynh/spyglass/2022.5.15/a5huynh.spyglass.installer.yaml:4]:
```suggestion
PackageIdentifier: athlabs.spyglass
```

Also modify the file name and folder name to match it.

File: manifests/a/a5huynh/spyglass/2022.5.15/a5huynh.spyglass.installer.yaml [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+PackageVersion: 2022.5.15
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: wix
+InstallerSwitches:
+  Silent: /qn
+  SilentWithProgress: /qb
```
Comment (KaranKad) [inline manifests/a/a5huynh/spyglass/2022.5.15/a5huynh.spyglass.installer.yaml:11]:
```suggestion
```

Switches are unnecessary

File: manifests/a/a5huynh/spyglass/2022.5.15/a5huynh.spyglass.locale.en-US.yaml [hunk 8]
```diff
@@ -0,0 +1,30 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS1.7-2-4
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: a5huynh.spyglass
```
Comment (KaranKad) [inline manifests/a/a5huynh/spyglass/2022.5.15/a5huynh.spyglass.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: athlabs.spyglass
```

File: manifests/a/a5huynh/spyglass/2022.5.15/a5huynh.spyglass.locale.en-US.yaml [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+# PrivacyUrl: 
+Author: Andrew Huynh
+PackageName: Spyglass
+PackageUrl: https://github.com/a5huynh/spyglass
+License: MIT
+LicenseUrl: https://github.com/a5huynh/spyglass/blob/v2022.5.15/LICENSE
+Copyright: Copyright (c) 2022 Andrew Huynh
+CopyrightUrl: https://github.com/a5huynh/spyglass/blob/v2022.5.15/LICENSE
+ShortDescription: Spyglass is an entirely local and configurable search engine, your computer crawls and indexes web addresses itself.
+Description: Spyglass is very much in its early stages, but it’s in a place where it's functional and can be used to replace basic searches. Spyglass is an entirely local and configurable search engine, your computer crawls and indexes web addresses itself.
```
Comment (KaranKad) [inline manifests/a/a5huynh/spyglass/2022.5.15/a5huynh.spyglass.locale.en-US.yaml:19]:
```suggestion
```

File: manifests/a/a5huynh/spyglass/2022.5.15/a5huynh.spyglass.locale.en-US.yaml [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+Moniker: spyglass
+Tags:
+- goggles
+- lenses
```
Comment (KaranKad) [inline manifests/a/a5huynh/spyglass/2022.5.15/a5huynh.spyglass.locale.en-US.yaml:23]:
```suggestion
```
These tags don't make sense

**Review by KaranKad — CHANGES_REQUESTED** [review]

**Comment by oparaskos** [comment]
> Hello @oparaskos,
> This pull request has been automatically marked as stale because it has been marked as requiring author feedback but has not had any activity for **7 days**. It will be closed if no further activity occurs **within 7 days of this comment**.

Boop

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Needs changes: https://github.com/microsoft/winget-pkgs/pull/61989#pullrequestreview-986458386

**Comment by KaranKad** [comment]
I've created a new pr at https://github.com/microsoft/winget-pkgs/pull/63378

**Pull request closed without merging**